packages/neon/neon/lib/src/utils/global_options.dart

```diff
@@ -232,5 +233,6 @@ class GlobalOptions {
 enum NavigationMode {
   drawer,
   drawerAlwaysVisible,
+  @Deprecated("The new design won't use this anymore")
```

Comment:
You can just remove it completely. That's the benefit of not having and promised stability ;)

Reply:
Not yet.
If someone (like my phone) has the navigation set to it it would fail when loading from storage.
I'll remove the entire setting in a followup.